include/podio/GenericParameters.h

```diff
@@ -140,15 +126,6 @@ class GenericParameters {
   friend void readGenericParameters(sio::read_device& device, GenericParameters& parameters, sio::version_type version);
 #endif
 
-#if PODIO_ENABLE_RNTUPLE
-  friend RNTupleReader;
-  friend RNTupleWriter;
-#endif
-
-#if !defined(__CLING__)
-  friend ROOTReader;
-#endif
-
```

Comment:
Same here, these changes if they are possible seem to be unrelated to the c++17 to c++20 changes (for me). If they are independent we should move them to another PR, I think.

Reply:
Moved to #706